### DISCUSSION TITLE
Speedify UI not working

### DISCUSSION THREAD

**Discussion by evanrodgers** [SmoothWAN/SmoothWAN#25] [General & Help]
Hello there,

I have the latest release of SmoothWAN running on an RPI 4 2GB and it's been working great for the last week or so. Curiously, today the Speedify UI stopped working. The internet does still work, but the UI does not respond. I've rebooted the RPI 4 and tested the web interface in Edge and Chrome. Any thoughts on what might be going on?

![image](https://user-images.githubusercontent.com/36175609/163686693-b80dc1a4-e4a0-4e2e-b17c-8ac4ee9127ec.png)

**Comment by TalalMash** [comment]
It seems like Speedify is not running, I'll release a new version soon, there is an issue with OpenWRT's rsync build.
Thanks for reporting, I'll convert it to an issue.

**Comment by TalalMash** [comment]
@evanrodgers Update released (0.99.6)
